tests/test_merge_db_functions.py

```diff
@@ -114,7 +114,7 @@ def test_load_master_list():
   master_file = "{0}/../utils/test_files/old_gfx90a68.HIP.fdb.txt".format(
       this_path)
 
-  basename = os.path.basename(master_file)
+  os.path.basename(master_file)
```

Comment:
earlier version, had to use this variable for testing purpose, we can remove as now basename is not being used.